assets/js/applicationinsights.js

```diff
@@ -9,13 +9,9 @@ window.applicationInsights = (function () {
         const clickPluginConfig = {
           autoCapture: true,
           callback: {
-            contentName: function (element) {
-              // If there is a id, use this as the content name
-              if (element.dataset.id) return element.dataset.id
-
-              // If this element is in the header or footer it could contain
-              // personal data so we use a default value instead
-              if (!element.closest('main')) return 'Unknown'
+            contentName: element => {
+              // If there is an id, use this as the content name, else return 'Unknown'
+              return element.dataset.id ? element.dataset.id : 'Unknown'
```

Comment:
The new linter did not like the fact that the function could (in theory) not have a return value (in practice it was never a problem)